backends/tty/tty.sh

```diff
@@ -1,20 +1,18 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
To allow other shells this should be:
```
#!/usr/bin/env sh
```

Reply:
However, `/bin/sh` is usually a symlink to another executable anyway.

Reply:
Hmm you're actually right. `/bin/sh` is fine :)